nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs

```diff
@@ -264,7 +264,8 @@ public static IEnumerable<Dependency> GetTopLevelPackageDependencyInfos(Immutabl
             var evaluationResult = GetEvaluatedValue(packageVersion, propertyInfo);
             if (evaluationResult.ResultType != EvaluationResultType.Success)
             {
-                throw new InvalidDataException(evaluationResult.ErrorMessage);
+                // if we can't resolve the package version, don't report the dependency
+                continue;
```

Comment:
Should we log anything? Or is it going to get logged by the Ruby code regardless?

Reply:
This doesn't necessarily mean anything so I don't think we need to log it.  We already log dependencies and properties that we could resolve, so if we're trying to figure out why something wasn't found, we can look there.